tests/fixtures/music_nerd_pro/beatles_prompt.txt

```diff
@@ -1 +0,0 @@
-Which band from the 60s wrote Yellow Submarine?
```

Comment:
This file is no longer used.
The spirit of this moved to the hocon test case here:
https://github.com/leaf-ai/neuro-san/blob/main/tests/fixtures/music_nerd/beatles_with_history.hocon